itest/lnd_misc_test.go

```diff
@@ -1271,44 +1270,6 @@ func testSignVerifyMessageWithAddr(ht *lntest.HarnessTest) {
 	require.False(ht, respValid.Valid, "external signature did validate")
 }
 
-// testNativeSQLNoMigration tests that nodes that have invoices would not start
-// up with native SQL enabled, as we don't currently support migration of KV
-// invoices to the new SQL schema.
-func testNativeSQLNoMigration(ht *lntest.HarnessTest) {
```

Comment:
but we still have a check in place that as soon as I start a boltdb version if the nativesql flag set, that it will fail with a detailed error that these two settings are not supported ?

Reply:
basically what we discussed personally we need to make sure we don't fall back to kv if we have migrated to native-sql for invoices.